.github/workflows/main.yml

```
@@ -0,0 +1,27 @@
+---
+name: Build
+
+on: 
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build
+        run: |
+          docker build -t bio-formats-octave .
+      - name: Prepare data
+        run: |
+          mkdir data
+          touch "data/test&sizeT=3.fake"
+          if [ ! -f data/tubhiswt-2D/tubhiswt_C0.ome.tif ]; then wget --user-agent GHA https://downloads.openmicroscopy.org/images/OME-TIFF/2016-06/tubhiswt-2D.zip && unzip tubhiswt-2D.zip && mv tubhiswt-2D data; fi 
```

```suggestion
          wget --user-agent GHA https://downloads.openmicroscopy.org/images/OME-TIFF/2016-06/tubhiswt-2D.zip && unzip tubhiswt-2D.zip && mv tubhiswt-2D data
```

assuming the caching is a next todo